piccolo/query/methods/alter.py

```diff
@@ -372,6 +390,20 @@ def rename_table(self, new_name: str) -> Alter:
         self._rename_table = [RenameTable(new_name=new_name)]
         return self
 
+    def rename_pk_constraint(
```

Comment:
I would change this to `rename_constraint`, as it could work for any kind of constraint.